src/components/Hero/Hero.css

```diff
@@ -0,0 +1,61 @@
+.Hero {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-areas: "hero-content hero-img";
+  height: 90vh;
+  width: 100%;
+  background: var(--bg-primary);
+  padding: 0 8rem;
+  position: relative;
+  z-index: 5;
+}
+.Hero .hero-img {
+  grid-area: hero-img;
+  display: flex;
+  align-items: center;
+  width: 50rem;
```

Comment:
you can try max-width so it can be responsive

Reply:
will fix in the next pr